From 68f4a2b1f859685f7cb66ebd6079b150d617a825 Mon Sep 17 00:00:00 2001
From: Ley Foon Tan <lftan@altera.com>
Date: Thu, 3 Apr 2014 14:02:22 +0800
Subject: [PATCH 228/248] FogBugz #194497: mailbox: Fixed compilation warning

Compilation warning when compiling for x86_64.
config: make ARCH=x86_64 allmodconfig
drivers/mailbox/mailbox-altera.c:337:3: warning:
format '%d' expects argument of type 'int', but
argument 3 has type 'long unsigned int' [-Wformat=]

v2:
- Change to use %zu

[Original patch taken from
git://git.rocketboards.org/linux-socfpga.git socfpga-3.14]

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Ley Foon Tan <lftan@altera.com>
---
 drivers/mailbox/mailbox-altera.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mailbox/mailbox-altera.c b/drivers/mailbox/mailbox-altera.c
index ddd16a5..5a055c1 100644
--- a/drivers/mailbox/mailbox-altera.c
+++ b/drivers/mailbox/mailbox-altera.c
@@ -332,7 +332,7 @@ static int altera_mbox_probe(struct platform_device *pdev)
 	mbox->ipc_con.ops = &altera_mbox_ops;
 
 	if ((strlen(np->name) + 1) > sizeof(mbox->ipc_con.controller_name))
-		dev_warn(&pdev->dev, "Length of mailbox controller name is greater than %d\n",
+		dev_warn(&pdev->dev, "Length of mailbox controller name is greater than %zu\n",
 		sizeof(mbox->ipc_con.controller_name));
 
 	ret = of_property_read_string(np, "linux,mailbox-name", &mbox_name);
-- 
1.9.1

